ios/Felfele.xcodeproj/project.pbxproj

```diff
@@ -1643,23 +1643,23 @@
 					"$(SRCROOT)/../node_modules/react-native-device-info/ios/RNDeviceInfo",
 					"$(SRCROOT)/../node_modules/react-native-background-fetch/ios/RNBackgroundFetch/**",
 				);
-				INFOPLIST_FILE = postmodern/Info.plist;
+				INFOPLIST_FILE = Felfele/Info.plist;
 				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
 				LIBRARY_SEARCH_PATHS = (
 					"$(inherited)",
-					"\"$(SRCROOT)/postmodern\"",
-					"\"$(SRCROOT)/postmodern\"",
-					"\"$(SRCROOT)/postmodern\"",
-					"\"$(SRCROOT)/postmodern\"",
+					"\"$(SRCROOT)/Felfele\"",
+					"\"$(SRCROOT)/Felfele\"",
+					"\"$(SRCROOT)/Felfele\"",
+					"\"$(SRCROOT)/Felfele\"",
 				);
 				OTHER_LDFLAGS = (
 					"$(inherited)",
 					"-ObjC",
 					"-lc++",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = "org.reactjs.native.example.$(PRODUCT_NAME:rfc1034identifier)";
-				PRODUCT_NAME = postmodern;
+				PRODUCT_BUNDLE_IDENTIFIER = org.felfele.mobile;
```

Comment:
I am not sure how it is supposed to work, but this used to include the `$PRODUCT_NAME` with an `rfc1034identifier` (whatever that means).